exasol/toolbox/tools/workflow.py

```diff
@@ -1,188 +1,70 @@
-import difflib
-import io
-from contextlib import ExitStack
 from pathlib import Path
-from typing import (
-    Any,
-    Mapping,
-    Union,
-)
-
-import importlib_resources as resources
 import typer
-from rich.columns import Columns
-from rich.console import Console
-from rich.syntax import Syntax
-
-stdout = Console()
-stderr = Console(stderr=True)
+from exasol.toolbox.tools import template
 
 CLI = typer.Typer()
-
-
-def _workflows() -> Mapping[str, Any]:
-    pkg = "exasol.toolbox.templates.github.workflows"
-
-    def _normalize(name: str) -> str:
-        name, _ = name.split(".")
-        return name
-
-    return {_normalize(w.name): w for w in resources.files(pkg).iterdir()}  # type: ignore
+PKG = "exasol.toolbox.templates.github.workflows"
+TEMPLATE_TYPE = "workflow"
+LEXER = "yaml"
 
 
 @CLI.command(name="list")
-def list_workflows(
+def list_issues(
     columns: bool = typer.Option(
         False, "--columns", "-c", help="use column style presentation like `ls`"
     )
 ) -> None:
-    """List all available workflows."""
-
-    class List:
-        def __init__(self, items: Any):
-            self.items = items
-
-        def __rich__(self) -> str:
-            return "\n".join(self.items)
-
-    output = List(_workflows()) if not columns else Columns(_workflows(), expand=True)
-    stdout.print(output)
+    """List all available issues."""
+    template.list_templates(columns=columns, pkg=PKG)
 
 
 @CLI.command(name="show")
-def show_workflow(
-    workflow: str = typer.Argument(..., help="Workflow which shall be shown."),
+def show_issue(
+    issue: str
```

Comment:
`ctrl + c` , `ctrl + v` error (Issue vs Workflow)?
Why did you remove the CLI help?